### ISSUE TITLE
Module build fails after updating to 0.5.0

### ISSUE THREAD

**Issue by boboldehampsink** [zimmo-be/twig-loader#50]
After updating to 0.5.0 module builds fail with the following message:

```
    Module build failed (from ./node_modules/twig-loader/index.js):
    TypeError: Cannot read property 'context' of undefined
        at Object.module.exports (/app/user/node_modules/twig-loader/lib/loader.js:25:63)
```

**Comment by webberig** [comment]
I'm guessing this is related to and fixed by #49 ?

**Comment by boboldehampsink** [comment]
I guess, since I'm using webpack 4 indeed :-) Let me verify

**Comment by webberig** [comment]
I released #49 as 0.5.1. If you still encounter issues, feel free to re-open!

**Comment by boboldehampsink** [comment]
It worked, thanks!
